pages/Elevator-Plate.md

```diff
@@ -0,0 +1,7 @@
+## Obtaining
+Elevator Plate can be crafted in an [Enchanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
```

Comment:
Might be splitting hairs here, but I feel like saying The Elevator Plate at the start of the sentence is a bit nicer.